tf2onnx/optimizer/__init__.py

```diff
@@ -6,43 +6,51 @@
 from __future__ import print_function
 from __future__ import unicode_literals
 
-import sys
-import traceback
 from collections import OrderedDict
+import copy
 
-from tf2onnx.optimizer.const_fold_optimizer import ConstFoldOptimizer
-from tf2onnx.optimizer.identity_optimizer import IdentityOptimizer
-from tf2onnx.optimizer.merge_duplicated_nodes_optimizer import MergeDuplicatedNodesOptimizer
-from tf2onnx.optimizer.transpose_optimizer import TransposeOptimizer
-
-# pylint: disable=missing-docstring, broad-except
+from .const_fold_optimizer import ConstFoldOptimizer
+from .identity_optimizer import IdentityOptimizer
+from .merge_duplicated_nodes_optimizer import MergeDuplicatedNodesOptimizer
+from .transpose_optimizer import TransposeOptimizer
+from .. import logging
 
 # optimizer sequence need to be considered carefully
 _optimizers = OrderedDict([
-    ("transpose_opt", TransposeOptimizer),
-    ("fold_const", ConstFoldOptimizer),
-    # merge_duplicated_nodes should be used after transpose_opt
-    # for transpose_opt may have some trans nodes that can be merge
-    ("merge_duplicated_nodes", MergeDuplicatedNodesOptimizer),
-    ("identity_opt", IdentityOptimizer),
+    ("reduce_transpose", TransposeOptimizer),
+    ("fold_constants", ConstFoldOptimizer),
+    # merge_duplication should be used after reduce_transpose
+    # for reduce_transpose may have some trans nodes that can be merge
+    ("merge_duplication", MergeDuplicatedNodesOptimizer),
+    ("reduce_identity", IdentityOptimizer),
```

Comment:
shall we call it remove_transpose/remove_identity?

Reply:
"remove" will let others assume we can remove all trans/identity?

Reply:
optimize_transpose, remove_identity?